### PR TITLE
The android example fixed

### PR DESCRIPTION
**Pull request by harrydema**
I took the job to link the library on android and fix the small errors a saw.

The example is now working on IOS and Android

### CODE REVIEW

File: Example/android/gradle.properties [hunk 1]
```diff
@@ -18,3 +18,4 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+android.enableAapt2=false
```
Comment (slycoder) [inline Example/android/gradle.properties]: This is no longer required as of RN 57 right?

File: Example/index.ios.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
+import React, { Component } from 'react';
```
Comment (slycoder) [inline Example/index.ios.js]: Do you know offhand the differences between android and ios?  They should hopefully be close to parity.

File: Example/android/gradle.properties (hunk 1 above)
Comment (harrydema) [reply Example/android/gradle.properties]: This proyect has react-native 0.44 as declared on the package.json

File: Example/index.ios.js (hunk 2 above)
Comment (harrydema) [reply Example/index.ios.js]: I replicated your index.js on index.android.js and index.ios.js as the index.js file was not found

File: Example/android/gradle.properties (hunk 1 above)
Comment (slycoder) [reply Example/android/gradle.properties]: Ah, I see.  Setting this potentially breaks other stuff if someone were to follow it.  If it works as is here in the example, it's fine.  But would be better if we could peg it to 0.57.x and remove this line.

File: Example/index.ios.js (hunk 2 above)
Comment (slycoder) [reply Example/index.ios.js]: That's weird.  It should find it if it's called index.js.

File: Example/android/gradle.properties (hunk 1 above)
Comment (harrydema) [reply Example/android/gradle.properties]: Someone should update the example to RN 0.57

File: Example/index.ios.js (hunk 2 above)
Comment (harrydema) [reply Example/index.ios.js]: That was what i thought, but it was unable to find it, so i had to change it

File: Example/android/gradle.properties (hunk 1 above)
Comment (slycoder) [reply Example/android/gradle.properties]: True dat, i'll open up a separate issue.

File: Example/index.ios.js (hunk 2 above)
Comment (harrydema) [reply Example/index.ios.js]: @slycoder can you approve the review?

Comment (slycoder) [reply Example/index.ios.js]: I'd really like to get to the bottom of this first.  If you're in a pinch, can you revert the index.js change, I'll merge it, and then try to repro locally and we can address the issue in a separate PR.

Comment (umarniz) [reply Example/index.ios.js:1]: This issue of `index.android` not found happens on certain android devices when `getJSMainModuleName()` is not defined, like defined in master now: https://github.com/blackuy/react-native-twilio-video-webrtc/blob/master/Example/android/app/src/main/java/com/example/MainApplication.java#L32

**Comment by umarniz** [comment]
@slycoder I think this PR can be closed as your example + my PR on top of it have fixed the android sample!

**Comment by slycoder** [comment]
Agreed.

**Pull request closed without merging**